### ISSUE TITLE
Potential unresponsiveness

### ISSUE THREAD

**Issue by jesus-mg-ios** [firebase/firebase-ios-sdk#14140]
### Description

<img width="1016" alt="image" src="https://github.com/user-attachments/assets/afa8f905-c3b1-4c17-beb5-743d40534ed5">

### Firebase SDK Version

10.25.0

### Xcode Version

16.1

### Installation Method

Swift Package Manager

### Firebase Product(s)

Crashlytics

### Targeted Platforms

iOS

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

**Comment by MichaelVerdon** [comment]
Does this issue occur on the latest sdk version?

**Comment by themiswang** [comment]
Issue is solved in #12913, update to latest firebase version should solve this. Gonna close this issue
